src/frontend/js/project-service.js

```diff
@@ -33,7 +33,8 @@ angular.module('seashell-projects', ['seashell-websocket', 'marmoset-bindings'])
       var SKEL_ROOT_URL = CS136_URL + "assignment_skeletons/";
       // TODO: update with real template path.
       var PROJ_ZIP_URL_TEMPLATE = SKEL_ROOT_URL + "%s-seashell.zip";
-      var PROJ_FILE_LIST_URL_TEMPLATE = CGI_URL + "skeleton_file_list.rkt?template=%s";
+      var USERNAME = $cookies.getObject(SEASHELL_CREDS_COOKIE).user;
```

Comment:
Where is the dummy cookie created? I can't seem to find it after a lot of searching